### PR TITLE
[DEVOPS-24]  scripts/aws.hs:  control S3 redirection of the Daedalus release bucket

### PR DESCRIPTION
**Pull request by deepfire**
This implements the scheme due to investigations that culminated in  https://issues.serokell.io/issue/DEVOPS-24#comment=96-3899

### CODE REVIEW

**Comment by domenkozar** [comment]
Looks good!

A few comments:

- the script should be in `scripts/aws.hs`
- by default it should point to installers version as listed in https://test.daedaluswallet.io/#download and be configured for bucket daedalus-travis
- last but not least, there should be docs at https://github.com/input-output-hk/internal-documentation/wiki/Daedalus describing how to update the redirection for a new given release

**Comment by deepfire** [comment]
@domenkozar, 

1. Shouldn't we move the `CardanoSL.hs`, `Infra.hs` etc. scripts there as well, then?
2. On it.
3. The docs were done, and are here:
    - https://github.com/input-output-hk/internal-documentation/wiki/Operational-Manual#bumping-of-httpdaedalus-installer-routing-ruless3-websiteeu-central-1amazonawscomdaedalus-ostype-latestpkext-links

**Comment by deepfire** [comment]
The URLs are now:

- http://daedalus-travis.s3-website.eu-central-1.amazonaws.com/daedalus-osx-latest.pkg
- http://daedalus-travis.s3-website.eu-central-1.amazonaws.com/daedalus-win64-latest.exe

The former is live, however the latter started complaining wrt. OCSP:

```
An error occurred during a connection to appveyorcidata.blob.core.windows.net. Invalid OCSP signing certificate in OCSP response. Error code: SEC_ERROR_OCSP_INVALID_SIGNING_CERT
```

File: AWS.hs [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  , key   :: T.Text
+  } deriving (Show)
+
+ppURL:: URL -> T.Text
+ppURL (URL pro ho ke) = pro <> "://" <> ho <> "/" <> ke
+
+

+-- * Buckets
+data Bucket     = Bucket { fromBucket ∷ T.Text } deriving (Show)
```
Comment (jmitchell) [inline AWS.hs]: Why the oddball unicode symbol `∷`? I notice the `UnicodeSyntax` extension is enabled too, but for the most part you're not using it.

My personal preference is avoiding features like this unless we make a style guide that explicitly encourages it. Consistency is good.

**Comment by jmitchell** [comment]
> the latter started complaining wrt. OCSP:

That's no good. I reproduced this in Firefox Nightly. It's a non-issue in Private Browsing mode though. Also works fine for me in Chromium.

File: AWS.hs (hunk 1 above)
Comment (deepfire) [inline AWS.hs]: I've changed the file to be entirely `UnicodeSymbols`, for consistency.

**Comment by deepfire** [comment]
Documentation updated to reflect the S3 bucket name change.

**Comment by domenkozar** [comment]
@deepfire where are the docs located?

**Comment by deepfire** [comment]
@domen, https://github.com/input-output-hk/internal-documentation/wiki/Operational-Manual#bumping-of-httpdaedalus-traviss3-websiteeu-central-1amazonawscomdaedalus-ostype-latestpkext-links

**Pull request merged**